src/overlays/actors/ovl_Item_Shield/z_item_shield.c

```diff
@@ -217,11 +239,15 @@ void ItemShield_Update(Actor* thisx, PlayState* play) {
 void ItemShield_Draw(Actor* thisx, PlayState* play) {
     ItemShield* this = (ItemShield*)thisx;
 
-    if (!(this->unk_19C & 2)) {
+    if (!(this->stateFlags & ITEMSHIELD_STATE_INVISIBLE)) {
         OPEN_DISPS(play->state.gfxCtx, "../z_item_shield.c", 457);
         Gfx_SetupDL_25Opa(play->state.gfxCtx);
         gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx, "../z_item_shield.c", 460),
                   G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        //! @bug This displaylist will attempt to draw a displaylist from segment 12, but this actor never sets segment
+        //! 12, so it will draw whatever was last set in segment 12. Therefore when that was not a displaylist, it is
```

Comment:
can possibly reword to remove "segment 12" being repeated 3 times in a row.
"This display list references segment 12, which the player actor usually sets up properly before drawing his shield. However when drawn in isolation by this actor, that segment was not set properly. If an actor that processes before this one sets segment 12 to something other than a display list, a crash is likely to occur. A notable example..."